.code-samples.meilisearch.yaml

```diff
@@ -3,13 +3,45 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
+async_guide_filter_by_date_1: |-
+  client.get_tasks(after_enqueued_at: DateTime.new(2020, 10, 11, 11, 49, 53))
+
+async_guide_multiple_filters_1: |-
+  client.get_tasks(index_uids: ['movies'], types: ['documentAdditionOrUpdate', 'documentDeletion'], statuses: ['processing'])
+
+async_guide_filter_by_ids_1: |-
+  client.get_tasks(uids: [5, 10, 13])
+
+async_guide_filter_by_statuses_1: |-
+  client.get_tasks(statuses: ['failed', 'canceled'])
+
+async_guide_filter_by_types_1: |-
+  client.get_tasks(types: ['dumpCreation', 'indexSwap'])
+
+async_guide_filter_by_index_uids_1: |-
+  client.get_tasks(index_uids: ['movies'])
+
+delete_tasks_1: |-
+  client.delete_tasks(uids: [1, 2])
+
+cancel_tasks_1: |-
+  client.cancel_tasks(uids: [1, 2])
+
+async_guide_canceled_by: |-
+  client.get_tasks(canceled_by: [9])
+
+swap_indexes_1: |-
+  client.swap_indexes([['indexA', 'indexB'], ['indexX', 'indexY']]);
+
+search_parameter_guide_hitsperpage_1: |-
+  client.index('movies').search('', hits_per_page: 15)
```

Comment:
Why do we sometimes have `_` in the code sample name and sometimes not? 
For example, for the `swap_indexes` method we have `swap_indexes_1` as code sample name, but here for the `hits_per_page` parameter we have `search_parameter_guide_hitsperpage_1`. I would have expected `search_parameter_guide_hits_per_page_1` instead 🤔

This is anyway not important at all so I'll approve the PR, I just wanted to know if there was some logic behind it that I don't get :)

Reply:
It is a good question, and I think this is an arbitrary name that the docs team chose (so maybe it is an error).